warehouse/query-core/src/main/java/datawave/query/QueryParameters.java

```diff
@@ -211,6 +211,11 @@ public class QueryParameters {
      */
     public static final String EXCERPT_FIELDS = "excerpt.fields";
 
+    /**
+     * Used to specify summaries that should be returned.
```

Comment:
Consider renaming to SUMMARY_SIZE. Doc should be updated to include the size unit and any default value (I think the default value is 150 below). Size in bytes? The number of words or tokens?

Reply:
fixed